pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -995,6 +1023,13 @@ public void addUnAckedMessages(int numberOfMessages) {
         topic.getBrokerService().addUnAckedMessages(this, numberOfMessages);
     }
 
+    @Override
+    public void afterAckMessages(Object position, Throwable error, Object ctx) {
+        if (!cursor.isMetadataTooLargeToPersist()) {
```

Comment:
@coderzc 

> Use shouldPauseOnAckStatePersist directly?

If we call `shouldPauseOnAckStatePersist(ReadType.Normal)` here, the param `ReadType.Normal` is confuse.


@codelipenghui 

> Will it waste too many CPU circles if cursor.isMetadataTooLargeToPersist() is always false? This method readMoreEntriesAsync will be executed by another thread and it will do lot of checks for each call.
> I think what we want to do is if the dispatcher is paused by the ack state persistence before, but now the ack state persistence is good. We should resume the message dispatching.

Good suggestion. I added a counter indicates how many times the dispatching has been paused, and call `readMoreEntries` only the counter is larger than `0`